src/util/getBeaconHandler.ts

```diff
@@ -1,8 +1,10 @@
 export abstract class GetBeaconHandler<R> {
     send(url: string): Promise<R> {
-        return fetch(url, { method: "GET", keepalive: true }).then((r) =>
-            this.makeSendResult(r),
-        )
+        return fetch(url, {
+            method: "GET",
+            // keepalive causes problems in some browsers
+            // keepalive: true
```

Comment:
I suspect it's only in CORS POST requests where keepalive is a problem, but I removed it here too, just in case.